### ISSUE TITLE
Google Search box not accessible (using JAWS or NVDA)

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#265]
Len Burns issue. I'm marking this Important because it needs review by Rob or Jeff. However, this is only for certain screen readers, so I assume it can be Could Wait if it's not an easy fix.

Len's text:
The search button associated with the search box displayed on each page is not readable via either Jaws or NVDA.  While it is possible to trigger the search by pressing enter, this is confusing.
There is no efficient way to located the results frame.  There is a level one heading at the site search, but the results appear below this.  Google itself places the results at a level 2 heading and each result at a level 3.  This allows for efficient skimming for the desired entry.
